src/onboarding/ChooseYourAdventure.tsx

```diff
@@ -0,0 +1,188 @@
+import Touchable from 'src/components/Touchable'
+import CeloIconNew from 'src/icons/CeloIconNew'
+import GraphSparkle from 'src/icons/GraphSparkle'
+import PlusIcon from 'src/icons/PlusIcon'
+import ProfilePlus from 'src/icons/ProfilePlus'
+import { nuxNavigationOptionsNoBackButton } from 'src/navigator/Headers'
+import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Shadow, Spacing } from 'src/styles/styles'
+import { walletAddressSelector } from 'src/web3/selectors'
+
+const AdventureCard = ({
+  onPress,
+  text,
+  index,
+  icon,
+}: {
+  onPress: () => void
+  text: string
+  index: number
+  icon: React.ElementType
+}) => (
+  <Card
+    key={index}
+    testID={`AdventureCard/${index}/${text}`}
+    style={styles.card}
+    rounded={true}
+    shadow={Shadow.BarShadow}
+  >
+    <Touchable style={styles.pressableCard} onPress={onPress}>
+      <>
+        <View style={styles.iconContainer}>{icon}</View>
+        <Text style={fontStyles.small500}>{text}</Text>
+      </>
+    </Touchable>
+  </Card>
+)
+
+// https://stackoverflow.com/a/2450976/112731
+// Fisher–Yates shuffle algorithm to shuffle an array
+function shuffle(array: any[], seed: string) {
+  let currentIndex = array.length,
+    temporaryValue,
+    randomIndex
+
+  // While there remain elements to shuffle...
+  while (0 !== currentIndex) {
+    // Pick a remaining element...
+    randomIndex = Math.floor(seedrandom(seed)() * currentIndex)
+    currentIndex -= 1
+
+    // And swap it with the current element.
+    temporaryValue = array[currentIndex]
+    array[currentIndex] = array[randomIndex]
+    array[randomIndex] = temporaryValue
+  }
+
+  return array
+}
+
+function ChoseYourAdventure() {
+  const { t } = useTranslation()
+  const address = useSelector(walletAddressSelector)
+
+  const getAdventureCards = () => {
+    const cardDetails = [
+      {
+        text: t('choseYourAdventure.options.add'),
+        goToNextScreen: () => {
+          navigateHome({ params: { initialScreen: Screens.FiatExchange } })
+        },
+        icon: <PlusIcon />,
+      },
+      {
+        text: t('choseYourAdventure.options.dapp'),
+        goToNextScreen: () => {
+          navigateHome({ params: { initialScreen: Screens.DAppsExplorerScreen } })
+        },
+        icon: <GraphSparkle />,
+      },
+      {
+        text: t('choseYourAdventure.options.profile'),
+        goToNextScreen: () => {
+          navigateHome()
+          navigate(Screens.Profile)
+        },
+        icon: <ProfilePlus />,
+      },
+      {
+        text: t('choseYourAdventure.options.learn'),
+        goToNextScreen: () => {
+          navigateHome({ params: { initialScreen: Screens.ExchangeHomeScreen } })
+        },
+        icon: <CeloIconNew />,
+      },
+    ]
+    return shuffle(cardDetails, address ?? Math.random().toString()).map(
```

Comment:
```suggestion
    return shuffle(cardDetails, address ?? DEFAULT_SEED).map(
```
(and then define default_seed somewhere)

I know this is a "should never happen" case, but if it happened anyway, I'd prefer that the user at least got a consistent result, rather than a different one each time this is rendered.